### PR TITLE
Format the README.md files

### PR DESCRIPTION
**Pull request by bkchr**

### CODE REVIEW

File: templates/minimal/README.md [hunk 1]
```diff
@@ -2,28 +2,29 @@
 
 # Polkadot SDK's Minimal Template
 
-<img height="70px" alt="Polkadot SDK Logo" src="https://github.com/paritytech/polkadot-sdk/raw/master/docs/images/Polkadot_Logo_Horizontal_Pink_White.png#gh-dark-mode-only"/>
+  * [ ] <img height="70px" alt="Polkadot SDK Logo" src="https://github.com/paritytech/polkadot-sdk/raw/master/docs/images/Polkadot_Logo_Horizontal_Pink_White.png#gh-dark-mode-only"/>
```
Comment (ggwpez) [inline templates/minimal/README.md:5]:
```suggestion
<img height="70px" alt="Polkadot SDK Logo" src="https://github.com/paritytech/polkadot-sdk/raw/master/docs/images/Polkadot_Logo_Horizontal_Pink_White.png#gh-dark-mode-only"/>
```
checkbox?

**Review by ggwpez — APPROVED** [review]
@alvicsam also need this markdown check required please 🙏

**Review by AndWeHaveAPlan — APPROVED** [review]

**Review by kianenigma — APPROVED** [review]

File: templates/minimal/README.md (hunk 1 above)
Comment (bkchr) [reply templates/minimal/README.md:5]: Fatfinger :D

**Comment by alvicsam** [comment]
@ggwpez sure, done

**Pull request merged**